src/components/BaseTextField/index.tsx

```diff
@@ -71,11 +71,34 @@ export interface BaseTextFieldProps extends BaseComponentProps {
   onMouseLeave?: ((event: React.MouseEvent<HTMLInputElement | HTMLTextAreaElement>) => void);
 }
 
+/**
+ * This is a hack to allow us to accept jsx elements in the description prop.
+ *
+ * We depend on onRenderDescription to display the description. However, the fabric
+ * textfield will only invoke it when it has a truthy description or errorMessage prop.
+ * Therefore we pass an arbitrary truthy string. It's value is only used to invoke
+ * onRenderDescription, and is otherwise ignored.
+ */
+const descriptionPresent = (props: BaseTextFieldProps) => {
+  if (props.description === undefined) {
+    return undefined;
```

Comment:
should this return false? as-is the function's return type is `true | undefined`

Reply:
The type is actually `string | undefined`, which is what it needs to be.

Maybe using the string `'true'` here is confusing, do you think something like `'ignored truthy string'` be better?

Reply:
oh duh, that's a string not a bool. eh it's fine, this can go away when we get a chance to update fabric